tests/httpUtils.spec.ts

```diff
@@ -0,0 +1,33 @@
+import { getURL } from '../src/utils/httpUtils';
+
+describe('getURL', () => {
+  it('should return base + path if not fedramp', () => {
+    const base = 'api.dev.snyk.io';
+    const path = '/analysis';
+
+    const result = getURL(base, path);
+
+    expect(result).toEqual(base + path);
+  });
+
+  it('should return base + org routing + path if fedramp', () => {
+    const base = 'api.snykgov.io';
+    const path = '/analysis';
+    const orgId = '1-2-3-4';
+
+    const result = getURL(base, path, orgId);
+
+    expect(result).toEqual(base + '/hidden/orgs/' + orgId + '/code' + path);
+  });
+
+  it('should throw an error if fedramp and org is missing', () => {
+    const base = 'api.snykgov.io';
+    const path = '/analysis';
+
+    try {
+      const result = getURL(base, path);
+    } catch (err) {
+      expect(err).toBeDefined();
+    }
```

Comment:
suggestion: let's make this test more specific, since there can be any other failure and we are not checking that the error thrown is the on we are expecting

```suggestion
    expect(() => getURL(base, path)).toThrowError('Org is required for this operation');
```